go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/oklog/run v1.1.0
 	github.com/openshift/api v3.9.0+incompatible
+	github.com/operator-framework/operator-lib v0.11.0
```

Comment:
What is this 3rd party dependency?

Reply:
Good catch. It was used to be able to set up proxy environmental variables if they were to be set, but we don't seem to be using that logic anywhere so I removed it.